tutorials/tutorial2.ipynb

```diff
@@ -470,44 +272,30 @@
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": 24,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "INFO:2020-06-10 18:23:37,143:AtacWorks-main] Checkng input files for compatibility\n",
-      "\u001b[33mBuilding model: resnet ...\u001b[0m\n",
-      "\u001b[33mLoading model weights from ./models/model.pth.tar...\u001b[0m\n",
-      "INFO:2020-06-10 18:23:46,923:AtacWorks-model_utils] Compiling model in DistributedDataParallel\n",
-      "INFO:2020-06-10 18:23:47,028:AtacWorks-model_utils] Compiling model in DistributedDataParallel\n",
-      "\u001b[33mFinished loading.\u001b[0m\n",
-      "INFO:2020-06-10 18:23:47,057:AtacWorks-model_utils] Compiling model in DistributedDataParallel\n",
-      "INFO:2020-06-10 18:23:47,155:AtacWorks-model_utils] Compiling model in DistributedDataParallel\n",
-      "\u001b[33mFinished building.\u001b[0m\n",
-      "Inference -------------------- [ 0/29] \n",
-      "\u001b[33mInference time taken:  184.018s (Load    6.425s,Prediction  177.351s)\u001b[0m\n",
-      "INFO:2020-06-10 18:26:57,449:AtacWorks-main] Waiting for writer to finish...\n",
-      "Writing the output to bigwig files\n"
-     ]
-    }
-   ],
    "source": [
+    "```\n",
     "!atacworks denoise \\\n",
-    "    --input_files NK.50_cells.h5 \\\n",
+    "    --noisybw dsc.1.NK.50.cutsites.smoothed.200.bw \\\n",
+    "    --interval_size 50000 \\\n",
     "    --genome $atacworks/data/reference/hg19.auto.sizes \\\n",
-    "    --config configs/infer_config.yaml \\\n",
-    "    --config_mparams configs/model_structure.yaml \\\n",
-    "    --threshold None"
+    "    --out_home \"./\" \\\n",
+    "    --exp_name \"ataworks_denoise_probs\" \\\n",
+    "    --distributed \\\n",
+    "    --threshold None\n",
```

Comment:
is `--threshold` a required command-line parameter? If not, we should probably ask users to modify it in the config file rather than from the command line, for consistency.